.pre-commit-config.yaml

```
@@ -68,7 +76,7 @@ repos:
         args:
           # Indent by two spaces
           - -i
-          - '2'
+          - "2"
```

Avoid making unnecessary changes from the skeleton:
```suggestion
          - '2'
```

---

@dav3r resolved with [b732ed3](https://github.com/cisagov/con-pca-tasks/pull/1/commits/b732ed37a0ef693f9c2844d6e84edfd78f2fb382)